x/service/types/relay.go

```diff
@@ -42,7 +34,7 @@ func (req RelayRequest) GetSignableBytesHash() ([32]byte, error) {
 
 	// return the marshaled request hash to guarantee that the signable bytes
 	// are always of a constant and expected length
-	return GetHashFromBytes(requestBz), nil
+	return protocol.GetHashFromBytes(requestBz), nil
```

Comment:
1. Thoughts on moving `GetHashFromBytes` into this file?
2. Should we rename it to `GetRelayHashFromBytes`?

Reply:
I explicitly moved it into the `pkg/crypto/protocol` package to consolidate shared (on/off-chain) mining-related static members.